torch/testing/_asserts.py

```diff
@@ -102,6 +97,63 @@ def wrapper(actual: Tensor, expected: Tensor, **kwargs: Any) -> Optional[Excepti
     return wrapper
 
 
+def _check_quantization(check_tensor_values: Callable[..., Optional[Exception]]) -> Callable[..., Optional[Exception]]:
+    """Decorates continuous tensor check functions to handle quantized tensors.
+
+    If the inputs are not quantized, this decorator is a no-op.
+
+    Args:
+        check_tensor_values (Callable[..., Optional[Exception]]): Tensor check function for continuous tensors.
+
+    Returns:
+        Optional[Exception]: Return value of :attr:`check_tensors`.
+    """
+
+    @functools.wraps(check_tensor_values)
+    def wrapper(actual: Tensor, expected: Tensor, **kwargs: Any) -> Optional[Exception]:
+        if not actual.is_quantized:
+            return check_tensor_values(actual, expected, **kwargs)
+
+        if actual.qscheme() == torch.per_tensor_affine:
```

Comment:
What happens if actual and expected have different quantization schemes?

Reply:
The test fails in `_check_attributes_equal`. See L209-L210.